examples/seismic/viscoacoustic/operators.py

```diff
@@ -183,7 +487,7 @@ def ForwardOperator(model, geometry, space_order=4, kernel='blanch_symes', save=
     eqn = eq_kernel(model, geometry, v, p, save=save)
 
     # The source injection term
-    src_term = src.inject(field=p.forward, expr=src * s)
+    src_term = src.inject(field=p.forward, expr=src * dt**2 / m)
```

Comment:
That factor depends on the PDE, the `dt**2/m` is only valid for 2nd order acoustic WE.

Reply:
Okay. We will fix it.